### ISSUE TITLE
Implement a DOM-based technique without using CNN as a baseline

### ISSUE THREAD

**Issue by marco-c** [marco-c/autowebcompat#195]
We should compare our technique based on CNN with a technique that doesn't use machine learning.

**Comment by sagarvijaygupta** [comment]
We should collect screenshots with DOM information also. Presently we don't have them.

**Comment by marco-c** [comment]
We have implemented collecting DOM information too, but we haven't collected any.

**Comment by Shashi456** [comment]
@marco-c do we simply need to run collect.py to start collecting data with dom info?

**Comment by marco-c** [comment]
Yes, I think so. We implemented it recently. You should run it for a couple of websites and check that it is actually generating correct data.

**Comment by Shashi456** [comment]
@marco-c although the dom info seems to be getting collected properly it is very slow in doing so. and could you tell me how i could add them to the repo since it's a git lfs file ?
